game/apps.py

```
@@ -1,5 +1,57 @@
+                    'client_id': client_id,
+                    'secret': secret,
+                }
+            )
+            if not created and (
+                app.client_id != client_id or app.secret != secret
+            ):
+                app.client_id = client_id
+                app.secret = secret
+                app.save()
+            app.sites.add(site)
+        except Exception as exc:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)